pyproject.toml

```
@@ -11,7 +11,7 @@ keywords = ['openapi', 'rest']
 
 [tool.poetry.dependencies]
 python = ">=3.7.0, <4.0.0"
-pyyaml = "^5.4"
+pyyaml = ">=5.4"
```

```suggestion
pyyaml = "^6.0"
```

I prefer to depend only on one major version, not both 5.4.x and 6.x.x. Two major versions can present breakinig changes.